content/20.pharmaceuticals.md

```diff
@@ -651,6 +651,19 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+###### Anakinra
+**Anticipated Mechanism.**
```

Comment:
```suggestion

```
We changed the format to less explicitly use these subheadings, so I'll remove them in the text here!